app/src/main/java/org/pesmypetcare/mypetcare/services/UserManagerAdapter.java

```diff
@@ -107,9 +108,18 @@ public void changeMail(String email, User user) {
     }
 
     @Override
-    public void createUser(String uid, String email, String password) {
+    public void createUser(String uid, String username, String email, String password) {
+        try {
+            ServiceLocator.getInstance().getUserManagerClient().createUser(uid, new UserData(username, email, password));
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Line is longer than 120 characters (found 121).](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5355440)